source/strings.tex

```diff
@@ -4836,6 +4872,22 @@
 Otherwise, returns \tcode{npos}.
 \end{itemdescr}
 
+\rSec2[string.view.deduction]{Deduction guide}
+
+\begin{itemdecl}
+template<class It, class End>
+  basic_string_view(It, End) -> basic_string_view<iter_value_t<It>>;
+\end{itemdecl}
+
+\begin{itemdescr}
+\pnum
+\constraints
+\begin{itemize}
+\item \tcode{It} satisfies \tcode{contiguous_iterator}, and
+\item \tcode{End} satisfies \tcode{sized_sentinel_for<It>}.
```

Comment:
As above.

Reply:
sized_sentinel_for takes two arguments, not just one.

@zygoloid, it seems novel to rely on the "prototype parameter" rule for a description like this. Can we fix this editorially or do we reject this for LWG cleanup or something else?

Reply:
Uh, the "novel" part is not actually true. We have in [ranges]:

> if C models derived_from&lt;forward_iterator_tag>

Reply:
Let revert this.

Reply:
Right. As you mention in #3456, there are quite a few such uses. It is actually in the concept definition and not their uses that the full spelling is the way to go when saying that the concept's parameters model it when A, B, and C.

Reply:
Similarly in `vformat_to`:

> `Out` satisfies `output_iterator<const charT\&>`.

Resolving this; I think this is consistent with what we do elsewhere. (Commas will be changed to periods as suggested.)